Sources/Purchasing/StoreKitAbstractions/PromotionalOffer.swift

```diff
@@ -122,11 +122,20 @@ extension PromotionalOffer.SignedData {
 
     @available(iOS 15.0, tvOS 15.0, watchOS 8.0, macOS 12.0, *)
     var sk2PurchaseOption: Product.PurchaseOption {
+        let signature: Data
+
+        if let decoded = Data(base64Encoded: self.signature) {
+            signature = decoded
+        } else {
+            // TODO: log warning
```

Comment:
Yeah, definitely worth logging something loud here. 
Stuff logged from here would also show up in the diagnostics API call, right? since we get offerings and parse products?

Reply:
This property is only used when purchasing though.